### DISCUSSION TITLE
What does "Java (🚧 coming soon): For Java environments, typically servers." mean

### DISCUSSION THREAD

**Discussion by naturzukunft** [m-ld/m-ld-spec#66] [Discussion]
Hi there,
are there somewhere more information about "Java (🚧 coming soon): For Java environments, typically servers."
Or is there already a project? Who will provide that implementation?

**Comment by gsvarovsky** [comment]
Hey! Thanks for popping by. The Java engine was actually the first we built... but it needs work to bring it to compliance with the current spec. So it's not published at the moment to avoid confusion.

Do you have a use-case in mind? It would be great to know the priority, and also whether anyone has capacity to help!

The tracker ticket is here: https://github.com/m-ld/m-ld-spec/issues/42

**Comment by naturzukunft** [comment]
hi, i answered the question here: https://socialhub.activitypub.rocks/t/querying-activitypub-collections/1866/7
let us stay there until we have concrete useCases.
